src/router/users.router.ts

```diff
@@ -1,12 +1,8 @@
-// Libraries
```

Comment:
Might be a good idea to add these comments to all the places for consistency? I'm already losing track of the proper order and sections (if they are supposed to even be a thing)

Reply:
Maybe just document it in some contributor recommendations/styleguide that will eventually be created?